src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color $time-smooth;
+
+    &:hover {
+      color: #00acdc;
```

Comment:
I see you fix the same problem for `$color-hover_link`. Will be nice to add variable here also